.azure-pipelines/third-party-test-suites.yml

```diff
@@ -0,0 +1,239 @@
+trigger: none
+pr: none
+
+schedules:
+- cron: "0 3 * * *"
+  displayName: Daily 3am (UTC) build
+  branches:
+    include:
+    - master
+  always: true
+
+# Global variables
+variables:
+  buildConfiguration: Release
+  dotnetCoreSdk5Version: 5.0.103
+  relativeTracerHome: /src/bin/windows-tracer-home
+  relativeArtifacts: /src/bin/artifacts
+  ddTracerHome: $(System.DefaultWorkingDirectory)/src/bin/dd-tracer-home
+  tracerHome: $(System.DefaultWorkingDirectory)/src/bin/windows-tracer-home
+  artifacts: $(System.DefaultWorkingDirectory)/src/bin/artifacts
```

Comment:
Aren't these moving to `$(System.DefaultWorkingDirectory)/bin/...` (dropping the `src`)

Reply:
I copied and pasted the values from our consolidated pipeline yml. I think you're right that the location has changed, but maybe we're no longer using some of these variables

Reply:
We should probably update the consolidated pipeline, but it's not a big issue. These variables override the "default" values, which, as you say, has dropped the src.